src/layer/vector/Canvas.js

```diff
@@ -355,26 +355,24 @@ export var Canvas = Renderer.extend({
 		}
 	},
 
-	_onMouseMove: function (e) {
-		if (!this._map || this._map.dragging.moving() || this._map._animatingZoom) { return; }
-
-		var point = this._map.mouseEventToLayerPoint(e);
-		this._handleMouseHover(e, point);
-	},
-
-
 	_handleMouseOut: function (e) {
 		var layer = this._hoveredLayer;
 		if (layer) {
 			// if we're leaving the layer, fire mouseout
 			DomUtil.removeClass(this._container, 'leaflet-interactive');
 			this._fireEvent([layer], e, 'mouseout');
 			this._hoveredLayer = null;
+			this._mouseHoverThrottled = false;
 		}
 	},
 
-	_handleMouseHover: function (e, point) {
```

Comment:
this change will break external plugins like VectorGrid
![image](https://user-images.githubusercontent.com/2462166/62537874-2c38d400-b85a-11e9-8c44-e54f4ee7b7b0.png)

Reply:
Damn, I wasn't familiar with those hacks in VectorGrid.

Will revert.